samples/StaticHtmlComposites/playwright.config.ts

```diff
@@ -0,0 +1,21 @@
+// playwright.config.ts
+import { PlaywrightTestConfig } from '@playwright/test';
+
+const config: PlaywrightTestConfig = {
+  // Put any shared options on the top level.
+  use: {
+    headless: true
+  },
+  retries: 1,
```

Comment:
Do we need retries here? i.e. is the instability forcing us to set this to `1`?

Reply:
Yes, there could be network issue here, which we mentioned on our standup, so give it one more chance to see if that passes

Reply:
Ah yep, can you add a comment here that we have this in to mitigate network issues
```suggestion
  // Add an extra retry to mitigate network issues.
  // This can be removed if we switch to using a mock ACS service.
  retries: 1,
```